Meson build

fepitre opened the pull request:
QubesOS/qubes-issues#6982

fepitre committed:
qubes-gui: fix implicit fallthrough

fepitre committed:
Switch to meson build

marmarek commented:
PipelineRetry

fepitre commented:
> PipelineRetry

Is there something to retry here? We may add a Gitlab job indeed.

marmarek commented:
I wasn't sure if the build was missing because it didn't worked, or isn't present in this repo at all. And that was the easiest method to check :)

jevank commented on gui-agent-qemu/qubes-gui.c:
```diff
@@ -503,6 +499,11 @@ static void qubesgui_message_handler(void *opaque)
                 case MSG_FOCUS:
                 case MSG_EXECUTE:
                     qs->vchan_data_to_discard = qs->hdr.untrusted_len;
```
I guess here is the reason https://github.com/QubesOS/qubes-issues/issues/7148 - default case should be one line over.

marmarek committed:
Fix handling unsupported messages

The body of an unsupported message needs to be skipped too, same as for
unknown ones.

Fixes QubesOS/qubes-issues#7148
